packages/mco/components/create-dr-policy/create-dr-policy.tsx

```diff
@@ -0,0 +1,237 @@
+  };
+
+  const [isOpen, setIsOpen] = React.useState(false);
+  const [selectedFormat, setSelectedFormat] = React.useState(
+    SyncSchedule.minutes
+  );
+
+  const setSyncSchedule = (time: string, format?: string) =>
+    setSchedule(`${time}${SCHEDULE_FORMAT[format ?? selectedFormat]}`);
+
+  const onSelect = (event) => {
+    const scheduleTime = schedule.match(/\d+/g)[0];
+    const newScheduleFormat = event.target.value;
+    setIsOpen(false);
+    setSelectedFormat(newScheduleFormat);
+    setSyncSchedule(scheduleTime, newScheduleFormat);
+  };
+
+  return (
+    <InputGroup>
+      <TextInput
+        id="sync-schedule"
+        defaultValue="5"
+        type="number"
+        onChange={(scheduleTime) => setSyncSchedule(scheduleTime)}
+        isRequired
+      />
+      <Dropdown
+        aria-label={t('Select schedule time format in minutes or seconds')}
+        onSelect={onSelect}
+        toggle={
+          <DropdownToggle onToggle={(open) => setIsOpen(open)}>
+            {selectedFormat}
+          </DropdownToggle>
+        }
+        isOpen={isOpen}
+        dropdownItems={[
+          <DropdownItem
+            key="minutes"
+            value={SyncSchedule.minutes}
+            component="button"
+          >
+            {SyncSchedule.minutes}
+          </DropdownItem>,
+          <DropdownItem
+            key="seconds"
+            value={SyncSchedule.seconds}
+            component="button"
+          >
+            {SyncSchedule.seconds}
+          </DropdownItem>,
+        ]}
+      />
+    </InputGroup>
+  );
+};
+
+type SyncScheduleProps = {
+  schedule: string;
+  setSchedule: React.Dispatch<React.SetStateAction<string>>;
+};
+
+export const CreateDRPolicy: React.FC<{}> = ({}) => {
+  const { t } = useTranslation('plugin__odf-console');
+  const history = useHistory();
+  const [policyName, setPolicyName] = React.useState('');
+  const [replication] = React.useState('');
+  const [selectedClusters, setSelectedClusters] = React.useState<Cluster[]>([]);
+  const [syncTime, setSyncTime] = React.useState('5m');
+
+  const onCreate = () => {
+    const payload: DRPolicyKind = {
+      apiVersion: getAPIVersionForModel(DRPolicyModel),
+      kind: DRPolicyModel.kind,
+      metadata: { name: policyName },
+      spec: {
+        drClusterSet: selectedClusters.map((cluster) => ({
+          name: cluster.name,
+          region: cluster.region,
+          s3profileName: '',
+        })),
+      },
+    };
+    if (replication === REPLICATION_TYPE(t)['async']) {
+      payload.spec.schedulingInterval = syncTime;
+    }
+    k8sCreate({ model: DRPolicyModel, data: payload });
+  };
+
+  return (
+    <div>
+      <PageHeading title={t('Create DRPolicy')}>
+        <TextContent className="mco-create-data-policy__description">
+          <Text component={TextVariants.small}>
+            {t(
+              'OpenShift provides a disaster recovery policy to allow quick recovery in a remote or secondary cluster.'
```

Comment:
```suggestion
              'OpenShift provides a disaster recovery policy to allow quick recovery in a remote or secondary cluster.'
```
There is some suggestion for this description on the marvelapp, please check with UX team once: https://marvelapp.com/prototype/hh948bh/screen/85462887